src/librustc_typeck/astconv.rs

```diff
@@ -2,6 +2,7 @@
 //! Conversion from AST representation of types to the `ty.rs` representation.
 //! The main routine here is `ast_ty_to_ty()`; each use is parameterized by an
 //! instance of `AstConv`.
+// ignore-tidy-filelength
```

Comment:
This commit should be merged into the one that pushed `astconv.rs` over the line limit.

Reply:
Actually, this seems to have already been addded: https://github.com/rust-lang/rust/issues/67418.